corehq/apps/hqwebapp/login_handlers.py

```diff
@@ -5,23 +5,26 @@
 
 from dimagi.utils.web import get_ip
 
+UNKNOWN_USER = 'unknown_user'
```

Comment:
Maybe use a value that has characters not allowed in a username?
```suggestion
UNKNOWN_USER = '<unknown>'
```
Even better would be to store no value at all (`None`/`null`) since the very definition of null is "unknown value" and it takes less space. That may require a database migration though, which may not be worth it, in which case I might still prefer an empty string to this.

Reply:
Handled in: c88f52ceddab0bbb9e2faaa6d8bd2923a41fa101